tests/unit/plot_sequence_logos/motif_HNRNPF_824

```diff
@@ -0,0 +1,9 @@
+//
+NA HNRNPF_824
+	A	C	G	U
```

Comment:
Now you have an inconsistency: here you have a `U` but in other input files (eg: `tests/unit/plot_sequence_logos/motif_HNRNPF_823`) you have a column with `T`. Please keep it with `T` for now and just adjust the plotting with another alphabet.

Reply:
I am not sure if I made myself clear: all the input files should stay exaclty the way they are - please just encode in your plotting script one flag to use "U" instead of "T".

Reply:
Done, I have added a piece of code which will do this for us!